README.md

```diff
@@ -1,59 +1,47 @@
-# Getting Started with Create React App
+# clockwork
+[![Contributors][contributors-shield]][contributors-link]
+[![Stargazers][stars-shield]][stars-link]
+[![Issues][issues-shield]][issues-link]
+[![Website][website-shield]][website-link]
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+clockwork is a website built to simplify the experience of searching courses and building timetables for the [University of Victoria](https://uvic.ca) (UVic). clockwork is a TypeScript app that uses React on the front-end and Firebase on the back-end.
```

Comment:
Clockwork in both sentences is lowercase, is this intentional?

Reply:
I thought it'd be cool aesthetics 😎  but I changed it